Dockerfile

```diff
@@ -61,6 +61,12 @@ wget -P /build/software/maven https://archive.apache.org/dist/maven/maven-2/2.2.
 
 RUN mkdir -p /build/software/java
 
+COPY OpenJDK11U-jdk_x64_linux_hotspot_11.0.4_11.tar.gz /build/software/java
```

Comment:
We can also direct download the OpenJDK 11 distributions via its GitHub releases page.

https://github.com/AdoptOpenJDK/openjdk11-binaries/releases/tag/jdk-11.0.4%2B11

May be we can do this for later openjdk versions...